dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Google’s postal address library that powering Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc.
+	2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip
+	2006-2013 Alexander Chemeris
+	2002 JSON.org"
```

Comment:
Congrats on making libaddressinput build with system rapidjson. Now that would also means that copyrights and licenses related to rapidjson could be dropped, as we're no longer using the in-tree version.